pkgs/development/python-modules/cppe/default.nix

```diff
@@ -0,0 +1,36 @@
+{ buildPythonPackage, lib, cmake, cppe, eigen
+, python, pybind11, numpy, h5py, numba, scipy
+, pandas, polarizationsolver, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  inherit (cppe) pname version src meta;
+
+  # The python interface requires eigen3, but builds from a checkout in tree.
+  # Using the nixpkgs version instead.
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace '"external/eigen3",' '"${eigen}/include/eigen3",'
+  '';
+
+  nativeBuildInputs = [
+    cmake
+    eigen
+  ];
+
+  dontUseCmakeConfigure = true;
+
+  buildInputs = [ pybind11 ];
+
+  checkInputs = [
+    numpy
+    h5py
+    numba
+    scipy
+    pandas
+    polarizationsolver
+    pytestCheckHook
```

Comment:
I generally prefer to sort these alphabetically, with ``pytestCheckHook`` at the top.